votor-messages/src/migration.rs

```diff
@@ -29,13 +29,13 @@
 //! Synchronization model:
```

Comment:
you can revert this entire file, it pulls in some in progress features - i'll clean it up when i do the next pr (actually integrating th emigration)